cli/src/generate/templates/package.json

```
@@ -4,6 +4,7 @@
   "description": "CAMEL_PARSER_NAME grammar for tree-sitter",
   "repository": "github:tree-sitter/tree-sitter-PARSER_NAME",
   "license": "MIT",
+  "types": "bindings/node/index.d.ts",
```

```suggestion
  "types": "bindings/node",
```
`index.d.ts` is the default.